polyfill/intersection-observer-test.js

```diff
@@ -941,6 +941,652 @@ describe('IntersectionObserver', function() {
 
   });
 
+  describe('same-origin iframe', function() {
```

Comment:
Can you move these tests inside the `observe()` describe block. The structure of the tests currently uses a single describe block per method, so I think it makes sense to stick with that unless you think these tests really don't fit that pattern.

Reply:
The tests are more orthogonal in this case and set up is very special for iframes and I wanted to reduce the spread across the board. But I also I wanted to make it concentrated to communicate "here are all the things that are special about iframes".

Reply:
OK. I still think it'd be nice if there were a way to logically group the same-document and iframe tests (maybe run all the tests in both scenarios), but since that would likely require a larger refactor, I don't want to block adding this feature.